src/codewhisperer/util/supplementalContext/supplementalContextUtil.ts

```diff
@@ -35,19 +35,18 @@ export async function fetchSupplementalContext(
     const timesBeforeFetching = performance.now()
     const dependencyGraph = DependencyGraphFactory.getDependencyGraph(editor.document.languageId)
 
-    if (dependencyGraph === undefined) {
-        // This is a general check for language support of CW.
-        // We perform feature level language filtering later.
-        return undefined
-    }
```

Comment:
makes dependency graph as optional since we are not actually utilizing all the APIs there